src/org/exist/dom/memtree/MemTreeBuilder.java

```diff
@@ -147,15 +147,15 @@ public int startElement(final QName qname, final Attributes attributes) {
 
             // parse attributes
             for(int i = 0; i < attributes.getLength(); i++) {
```

Comment:
probably we could use another loop, without the "i" index. A simple for loop over the attributes?